Identical packages in opentracing-api and opentracing-noop cause build failure with JDK 9 + jigsaw

Maven dependency on opentracing-util required to use GlobalTracer.get() introduced transitive dependency on opentracing-noop and opentracing-api.
Unfortunately both opentracing-noop and opentracing-api contains the same package "io.opentracing" and this causes error in JDK 9 + jigsaw projects containing module-info.java.

[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] module <module_name> reads package io.opentracing from both opentracing.api and opentracing.noop
[ERROR] module  reads package io.opentracing from both opentracing.noop and opentracing.api
[ERROR] module opentracing.api reads package io.opentracing from both opentracing.noop and opentracing.api
[ERROR] module opentracing.util reads package io.opentracing from both opentracing.noop and opentracing.api
[ERROR] module opentracing.noop reads package io.opentracing from both opentracing.noop and opentracing.api

It would be good to rename "io.opentracing" package in opentracing-noop to something like "io.opentracing.noop"

--- a/opentracing-mock/src/main/java/io/opentracing/mock/MockTracer.java
+++ b/opentracing-mock/src/main/java/io/opentracing/mock/MockTracer.java
@@ -16,7 +16,7 @@ package io.opentracing.mock;
 import io.opentracing.ActiveSpan;
 import io.opentracing.ActiveSpanSource;
 import io.opentracing.BaseSpan;
-import io.opentracing.NoopActiveSpanSource;
+import io.opentracing.noop.NoopActiveSpanSource;
 import io.opentracing.References;
 import io.opentracing.Span;
 import io.opentracing.SpanContext;

--- a/opentracing-noop/src/main/java/io/opentracing/noop/NoopActiveSpanSource.java
+++ b/opentracing-noop/src/main/java/io/opentracing/noop/NoopActiveSpanSource.java
@@ -11,7 +11,12 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing;
+package io.opentracing.noop;
+
+import io.opentracing.ActiveSpan;
+import io.opentracing.ActiveSpanSource;
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
 
 import java.util.Map;
 

--- a/opentracing-noop/src/main/java/io/opentracing/noop/NoopSpan.java
+++ b/opentracing-noop/src/main/java/io/opentracing/noop/NoopSpan.java
@@ -11,7 +11,10 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing;
+package io.opentracing.noop;
+
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
 
 import java.util.Map;
 

--- a/opentracing-noop/src/main/java/io/opentracing/noop/NoopSpanBuilder.java
+++ b/opentracing-noop/src/main/java/io/opentracing/noop/NoopSpanBuilder.java
@@ -11,7 +11,13 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing;
+package io.opentracing.noop;
+
+import io.opentracing.ActiveSpan;
+import io.opentracing.BaseSpan;
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
 
 import java.util.Collections;
 import java.util.Map;

--- a/opentracing-noop/src/main/java/io/opentracing/noop/NoopSpanContext.java
+++ b/opentracing-noop/src/main/java/io/opentracing/noop/NoopSpanContext.java
@@ -11,7 +11,9 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing;
+package io.opentracing.noop;
+
+import io.opentracing.SpanContext;
 
 import java.util.Collections;
 import java.util.Map;

--- a/opentracing-noop/src/main/java/io/opentracing/noop/NoopTracer.java
+++ b/opentracing-noop/src/main/java/io/opentracing/noop/NoopTracer.java
@@ -11,8 +11,12 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing;
+package io.opentracing.noop;
 
+import io.opentracing.ActiveSpan;
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
 import io.opentracing.propagation.Format;
 
 public interface NoopTracer extends Tracer {

--- a/opentracing-noop/src/main/java/io/opentracing/noop/NoopTracerFactory.java
+++ b/opentracing-noop/src/main/java/io/opentracing/noop/NoopTracerFactory.java
@@ -11,7 +11,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing;
+package io.opentracing.noop;
 
 public final class NoopTracerFactory {
     

--- a/opentracing-util/src/main/java/io/opentracing/util/GlobalTracer.java
+++ b/opentracing-util/src/main/java/io/opentracing/util/GlobalTracer.java
@@ -14,8 +14,8 @@
 package io.opentracing.util;
 
 import io.opentracing.ActiveSpan;
-import io.opentracing.NoopTracer;
-import io.opentracing.NoopTracerFactory;
+import io.opentracing.noop.NoopTracer;
+import io.opentracing.noop.NoopTracerFactory;
 import io.opentracing.Span;
 import io.opentracing.SpanContext;
 import io.opentracing.Tracer;
@@ -60,7 +60,7 @@ public final class GlobalTracer implements Tracer {
      * <p>
      * All methods are forwarded to the currently configured tracer.<br>
      * Until a tracer is {@link #register(Tracer) explicitly configured},
-     * the {@link io.opentracing.NoopTracer NoopTracer} is used.
+     * the {@link io.opentracing.noop.NoopTracer NoopTracer} is used.
      *
      * @return The global tracer constant.
      * @see #register(Tracer)

--- a/opentracing-util/src/test/java/io/opentracing/util/GlobalTracerTest.java
+++ b/opentracing-util/src/test/java/io/opentracing/util/GlobalTracerTest.java
@@ -13,8 +13,8 @@
  */
 package io.opentracing.util;
 
-import io.opentracing.NoopSpanBuilder;
-import io.opentracing.NoopTracerFactory;
+import io.opentracing.noop.NoopSpanBuilder;
+import io.opentracing.noop.NoopTracerFactory;
 import io.opentracing.SpanContext;
 import io.opentracing.Tracer;
 import io.opentracing.propagation.Format;

